src/styles/theme.ts

```diff
@@ -0,0 +1,201 @@
+const white = '#FFF'
+const blue = '#218DFF'
+const blueDark = '#0b66c6'
+const bgLight = '#edf2f7'
+const bgDark = '#2e2e2e'
+const blackLight = '#181a1b'
+// const black = '#000'
+
+export enum ThemeType {
+  MAIN = 'main',
+  SECONDARY = 'secondary',
+}
+
+interface ColourThemeBase {
+  [t: string]:
+    | {
+        [key in ThemeType]?:
+          | {
+              [prop: string]: string
+            }
```

Comment:
This can be `Record<string, string>`

Reply:
All these types can be Records, but maybe they would become less readable.
Up to you

Reply:
Yeah I opted for prop string as a more readable choice